raiwidgets/tests/responsibleai_dashboard/test_responsibleai_dashboard_input_erroranalysis.py

```diff
@@ -119,6 +137,22 @@ def test_rai_dashboard_input_adult_debug_ml_failure(
             flask_server_prediction_output,
             "Failed to generate json tree representation,")
 
+        # change metric back to be valid
+        metric = "Error rate"
+        # test invalid features throw errors
+        invalid_features = ['Income', 'InvalidFeature']
+        post_data = [invalid_features, filters, composite_filters,
+                     max_depth, num_leaves, min_child_samples, metric]
+        dashboard_input = ResponsibleAIDashboardInput(ri)
+
+        flask_server_prediction_output = dashboard_input.debug_ml(post_data)
+        self.check_failure_criteria(
+            flask_server_prediction_output,
+            "Failed to generate json tree representation,")
+        self.check_failure_criteria(
+            flask_server_prediction_output,
+            "not found in dataset. Existing features")
```

Comment:
Consider adding another test case for this scenario. Overloading tests with more code make it difficult to understand and update the test.

Reply:
done!